features/step_definitions/aspects_steps.rb

```diff
@@ -82,6 +82,12 @@ def aspect_dropdown_visible?
   click_aspect_dropdown
 end
 
+When /^(.*) in the aspect creation modal$/ do |action|
+  within('#newAspectModal') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.